BYT/ColorScheme.swift

```diff
@@ -94,8 +98,8 @@ class ColorScheme {
             let version_availableArrayOfDict = dict["version_available"] as? [[String: Any]]{
             var v = [VersionInColorScheme]()
             for dict in version_availableArrayOfDict {
```

Comment:
That was there from when Ana created the colorScheme... thought it may be necessary with version management.